SampleApp/Sources/PostsViewController.swift

```diff
@@ -43,14 +43,6 @@ final class PostsViewController: UIViewController {
         blueprintView.element = element
     }
 
-    private var viewSafeAreaInsets: UIEdgeInsets {
```

Comment:
This probably broke the tutorial build. Also a good time to make sure these build as part of the SampleApp!

Reply:
Nope, it didnt; wasnt used! But added tutorials as part of SampleApp build.